vendor-extra/JatsContentBundle/src/ViewConverter/FrontContentHeaderVisitor.php

```diff
@@ -31,13 +31,19 @@ protected function doVisit(Element $object, View $view, array &$context = []) :
 
         $title = $xpath->firstOf('jats:article-meta/jats:title-group/jats:article-title', $object);
 
+        $categories = $xpath->firstOf('jats:article-meta/jats:article-categories/jats:subj-group[@subj-group-type = "heading"]/jats:subject/ancestor::jats:article-categories', $object);
```

Comment:
Raises the interesting question about how granular each of these visitors are. Thinking that they should each done one thing, so they can easily be turned off/replaced individually (maybe you want to do something else completely with the categories list). So this one should be renamed `FrontArticleTitleContentHeaderVisitor`, and this part moved to `FrontSubjectGroupContentHeaderVisitor` (no need for the backtracking).

Reply:
I will rename `FrontArticleTitleContentHeaderVisitor` as a followup to this?

Reply:
Yeah, or can get it in now.